### DISCUSSION TITLE
I can't start termux via automation app

### DISCUSSION THREAD

**Discussion by creeper451** [termux/termux-app#3333] [Q&A] [answered]
I am attempting to start Termux as a background service through intents using the app available on F-Droid called "Automation" "https://f-droid.org/packages/com.jens.automation2/" I prefer not to use Tasker due to its proprietary nature. To accomplish this, I edited the Android manifest.xml file to include the necessary permissions for running commands and enabled it in the Termux properties. Despite these modifications, the intended functionality is not working.

**Comment by agnostic-apollo** [comment]
Without knowing what changes you have made and how you are attempting to start termux and what's the purpose of "start Termux as a background service", we can't help. You should also check logcat logs for issues. You can use `RUN_COMMAND` intent to start background commands, the service will automatically end when the command/script ends.

https://github.com/termux/termux-app#Debugging

https://github.com/termux/termux-app/wiki/RUN_COMMAND-Intent

**Reply by creeper451** [reply]
I apologize for not being clear earlier. My intention is to run commands directly from automation. I have already gone through the documentation you recommended earlier, but I faced permission issues while trying to start Termux. To resolve this, I added "com.termux.permission.RUN_COMMAND" to the automation app's "AndroidManifest.xml" file, and the permission was detected successfully. However, when I tried using the "Start Another Program" action and followed the instructions in the documentation, I still couldn't get it to work.

**Reply by agnostic-apollo** [reply, marked as answer]
>However, when I tried using the "Start Another Program" action and followed the instructions in the documentation, I still couldn't get it to work.

Again you have  not specified how you actually configured it and what documentation you followed. I don't use the app myself, but my "guess" is that you followed https://server47.de/automation/examples_startProgram.html but that does not have an option to start a service and only activity or broadcast, which is not gonna work since `RUN_COMMAND` intent is meant to be sent to `RunCommandService`.
